src/testRunner/runner.ts

```diff
@@ -209,12 +209,6 @@ namespace Harness {
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.ShimsWithPreprocess));
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.Server));
             // runners.push(new GeneratedFourslashRunner());
-
-            // CRON-only tests
-            if (process.env.TRAVIS_EVENT_TYPE === "cron") {

```

Comment:
You should keep this. Yes, I know it says "TRAVIS_EVENT_TYPE", but we replicated the environment variable, verbatim, into our cron-based devops jobs :V